src/components/RestaurantDetailedPage.js

```diff
@@ -24,11 +32,11 @@ export default function RestaurantDetailedPage() {
         />
       </header>
       <main>
-        <h1 className="detailed-heading">I Vigneri</h1>
+        <h1 className="detailed-heading">{singleRestaurant.name}</h1>
         <div className="detailed-subheading">
-          <h2>italian</h2>
+          <h2>{singleRestaurant.kitchen[0]}</h2>
```

Comment:
You're gonna want to iterate over this somehow, so you can display all the kitchen values, rather than just the first one

Reply:
For now its actually wanted that way as i have up to 4 definitions of cuisine in my json and that kind of ruins my css atm haha